sensor_msgs/PointCloud2[] clouds
geometry_msgs/Vector3[] robot_positions
int32[] classes
